packages/sdk-react/src/components/aiAssistant/utils/tableSort.ts

```
@@ -0,0 +1,135 @@
+
+  const parsedDate = getParsedDate(trimmedValue);
+  const isValidDate = parsedDate && isValid(parsedDate);
+
+  if (isValidDate) {
+    return parsedDate;
+  }
+
+  const numeric = trimmedValue.replace(/[^\d.-]/g, '').replace(/,/g, '');
+  const parsed = parseFloat(numeric);
+  const isValidNumber =
+    !isNaN(parsed) && isFinite(parsed) && numeric.length > 0;
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)